document/CoreFormat.mdk

```diff
@@ -1880,7 +1978,7 @@ A `rule` object contains information that describes a rule.
 
 Either the `shortDescription` property (&sect;[#rule-shortDescription])
 or the `fullDescription` property (&sect;[#rule-fullDescription])
-or both must be present.
+or both shall be present.
```

Comment:
We whacked `result.shortMessage`. This is `rule.shortDescription` (which I still like).